helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml

```diff
@@ -87,6 +87,11 @@ spec:
           memory: "16Gi"
 {{- end }}
 
+{{- if .Values.jaeger.esIndexCleaner }}
```

Comment:
The ES cleaner spec https://godoc.org/github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1#JaegerEsIndexCleanerSpec

A side effect approach is that some configuration can be provided in smcp CR and some in Jaeger CR. For example here users specify scheduled time but retention day can be specified in Jaeger CR.

Reply:
Not sure I follow - this template is used to generate the Jaeger CR, so it shouldn't be possible for the retention day to be specified unless it is defined in the smcp CR?

Reply:
The template is used to generate Jaeger CR, however the fields not specified in smcp CR can be directly edited in Jaeger CR and maistra operator would not change them bc it does not know about them. The smcp CR does not define what fields cleaner CR contains, it is a free form anything can be added there.

At least I remember this behavior from previous work on this CR. 

Also maybe condition `{{- if .Values.jaeger.esIndexCleaner }}` is redundant and cleaner could be passed there directly.